### PR TITLE
In dap-launch-find-launch-json, check if it exists before returning

### PR DESCRIPTION
**Pull request by nbfalcon**
Fixes issue (#335).

### CODE REVIEW

**Comment by nbfalcon** [comment]
I got the issue wrong in the commit-message, though :(

**Comment by nbfalcon** [comment]
Actually, wait, dont merge! It doesnt work.

File: dap-launch.el [hunk 1]
```diff
@@ -29,8 +29,10 @@
 
 (defun dap-launch-find-launch-json ()
   "Return the location of the launch.json file in the current project."
-  (when-let ((project (lsp-workspace-root)))
-    (concat project "/launch.json")))
+  (when-let ((project (lsp-workspace-root))
```
Comment (ericdallo) [inline dap-launch.el:32]: I think this should be:
```suggestion
  (-when-let* ((project (lsp-workspace-root))
```

Comment (nbfalcon) [reply dap-launch.el:32]: No. when-let allows that. If you dig trough its source, you will find a reference to if-let*, being defined as (let*).

**Comment by nbfalcon** [comment]
The last one works. I tested it by renaming my launch.json, after which its configurations didn't show up in dap-debug, and without renaming it. When not renaming it, its configuration showed up correctly.

**Comment by ericdallo** [comment]
Thank you!

**Pull request merged**